scripts/launch-s0-instance.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+me=$(basename $0)
+dir=$(readlink -f $(dirname $0))
+elastio_aws_id=537513441174
+security_group="elastio-s0-server"
+default_instance_type="m5d.2xlarge"
+default_instance_name="$(whoami)-s0-server"
+
+usage()
+{
+    echo "Usage examples:"
+    echo "   $me --key-name my_ec2_key --name my_s0_server --type m5d.2xlarge"
+    echo "   $me --k /path/to/key.pem -n my_s0_server -t m5d.2xlarge"
```

Comment:
This is misleading.  the key name isn't a path on the filesystem, it's the name assigned to the key pair in the EC2 console

Reply:
You are right. There is a kind of key name in the example with the long keys line above. Fixed.